packages/kbn-router-to-openapispec/src/generate_oas.ts

```diff
@@ -32,18 +32,19 @@ export interface GenerateOpenApiDocumentOptions {
   docsUrl?: string;
   tags?: string[];
   filters?: GenerateOpenApiDocumentOptionsFilters;
+  buildFlavor: string;
 }
 
 export const generateOpenApiDocument = (
   appRouters: { routers: Router[]; versionedRouters: CoreVersionedRouter[] },
   opts: GenerateOpenApiDocumentOptions
 ): OpenAPIV3.Document => {
-  const { filters } = opts;
+  const { filters, buildFlavor } = opts;
   const converter = new OasConverter();
   const getOpId = createOperationIdCounter();
   const paths: OpenAPIV3.PathsObject = {};
   for (const router of appRouters.routers) {
-    const result = processRouter(router, converter, getOpId, filters);
+    const result = processRouter(router, converter, getOpId, buildFlavor, filters);
```

Comment:
We'll have to do the same for `processVersionedRouter` below. Luckily `extractAuthzDescription` is already factored into a function so should be straight forward :)

Reply:
Done in https://github.com/elastic/kibana/pull/197001/commits/0437c30dda12a75ff7856fb1826c7326c1bf12b7